test/specs/orgBrowser.e2e.ts

```diff
@@ -112,12 +112,14 @@ describe('Org Browser', async () => {
     // Check MyClass is present under Apex Classes section
     const workbench = await (await browser.getWorkbench()).wait();
     await utilities.runCommandFromCommandPrompt(workbench, 'View: Show Org Browser', 5);
-    const refreshButton = await (
+    const refreshComponentsButton = await (
```

Comment:
Oh I see what's going on now!  Yes you're right about the topmost "Refresh Types" button not performing the same function as the "Refresh Components" button for the "Apex Classes" section.  Probably the "Refresh Components" button is enough but still it's safer to click both. 🙂

Interestingly, you previously had "Refresh Components" twice, so I do wonder why the duplicate used to be necessary.  But now with this code change I guess that question will have to go unanswered.

Reply:
Got it!  Yes confirming that the dialog is hiding the class name by manually following this step on my project. 👍